src/igvfd/schemas/in_vitro_system.json

```diff
@@ -176,6 +176,7 @@
         },
         "pooled_from": {
             "items": {
+                "title": "Biosample Pooled From",
```

Comment:
would not the title come from the Biosample?

Reply:
You're right, changed.